### PR TITLE
Gives a special *deathgasp text to IPC's.

### PR DESCRIPTION
**Pull request by Spacemanspark**
-Gives IPC's a different *deathgasp text. Idea taken from Baystation.
~Was also going to add a special text for when they get knocked out ("[name] encounters a hardware fault and suddenly reboots!"[an idea also taken from Baystation]) but the way we handle our emote code is different from Bay, apparently.
~Tested and works, both when using the emote manually and when killed.

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
>implying a machine was ever alive to begin with.

BAY LOGIC.

**Comment by SamCroswell** [comment]
Probably would be a good idea to generalize this.
IE: Have a deathgasp text var in the species datum and then have this check that var.

**Comment by NTSAM** [comment]
<3

**Comment by Markolie** [comment]
Well, emote files are already the biggest snowflakes around, so why not, I guess.

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -334,8 +334,12 @@
 					m_type = 2
 
 		if ("deathgasp")
-			message = "<B>[src]</B> seizes up and falls limp, \his eyes dead and lifeless..."
-			m_type = 1
+			if (species.name == "Machine")
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/emote.dm]: nice SNOWFLAKE CODE

**Comment by KasparoVy** [comment]
@SamCroswell The way it's implemented here, it'd be as simple as adding another if (species.name == "whatever") for the other species thankfully :)

**Comment by Iamgoofball** [comment]
NO, GOD NO

that's FUCKING TERRIBLE

**Comment by Spacemanspark** [comment]
:^)

**Pull request merged**